deployment/helm/ditto/values.yaml

```diff
@@ -169,8 +169,15 @@ dbconfig:
 ingress:
   # enabled whether Ingress should be enabled as alternative to the contained nginx
   enabled: false
+  # namespace for ingress controller, must exist
+  namespace: ingress-nginx
+  # Ingress-NGINX version
+  nginxIngressVersion: v1.8.0
+  # Nginx Version
+  nginxVersion: "1.21.6"
```

Comment:
This `nginxVersion` is already quite old.
Can we use a recent one?

Reply:
I followed the supported version table from https://github.com/kubernetes/ingress-nginx but probably a newer one should also work fine

Reply:
> I followed the supported version table from https://github.com/kubernetes/ingress-nginx but probably a newer one should also work fine

Ah, I see .. then I would also stick to that version 👍

Reply:
Ok, the suggested version than stays as default. The version table link is now included as description for the variables.